src/components/CEnqueueForm.vue

```diff
@@ -53,61 +52,69 @@
-            id="field_kwargs"
-            v-model.trim="message.kwargs"
-            @blur="checkForm"
-            placeholder='{"nums":[1,2,5]}'
-          ></textarea>
-          <small v-if="invalidKwargs" class="text-xs text-red-500">Invalid Kwargs</small>
-        </div>
-
-        <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
-          <label
-            class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
-            for="field_options"
-          >
+      <div>
+        <c-actor-argument
+          v-for="arg in selectedActor.args"
+          :name="arg.name"
+          :type="arg.type"
+          v-model="message.kwargs[arg.name]"
+          :key="arg.name"
+        />
+        <div class="w-full md:w-full mb-6 md:mb-0" v-if="true">
+          <label class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2">
             Options
           </label>
-          <textarea
-            :class="{ 'border-red-500': invalidOptions }"
-            class="appearance-none  h-40 block w-full bg-gray-200 focus:bg-white text-gray-700 border  rounded py-3 px-4 leading-tight focus:outline-none focus:border-gray-500"
-            id="field_options"
-            v-model.trim="message.options"
-            @blur="checkForm"
-            placeholder='{"isOn":true}'
-          ></textarea>
-          <small v-if="invalidOptions" class="text-xs text-red-500">Invalid Options</small>
+          <form class="w-full md:w-full h-10 flex justify-between mb-3" @submit.prevent="addOption">
+            <select
+              class="block appearance-none mr-1 w-1/2 h-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+              v-model="selectedOption"
+              @change="selectOption"
+              required
+            >
+              <option v-for="(type, option) in filteredOptions" :key="option" :value="option">
+                {{ option }} : {{ type }}
+              </option>
+            </select>
+            <input
+              :class="
+                filteredOptions[selectedOption] === 'bool'
+                  ? 'w-10 mx-10'
+                  : 'appearance-none block mx-1 h-full w-1/2 mb-4 bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500'
+              "
+              v-model="optionValue"
+              :type="
+                filteredOptions[selectedOption] === 'int'
+                  ? 'number'
+                  : filteredOptions[selectedOption] === 'bool'
+                  ? 'checkbox'
+                  : 'text'
+              "
+              :required="filteredOptions[selectedOption] !== 'bool'"
+            />
+            <button
```

Comment:
I feel this Add / Remove logic could be abstracted in a component, using slots https://vuejs.org/v2/guide/components.html#Content-Distribution-with-Slots